packages/terra-clinical-data-grid/src/DataGrid.module.scss

```diff
@@ -10,42 +10,48 @@
 
```

Comment:
The variable `terra-clinical-data-grid-pinned-column-box-shadow` is used in two different places with different default styles (one has an inset).

Reply:
![https://media.giphy.com/media/3o7TKNFKvLdpelkwfK/giphy.gif](https://media.giphy.com/media/3o7TKNFKvLdpelkwfK/giphy.gif)

Reply:
Double checked with Neil, the separate values are intended. So I renamed the variable for the scrollbar to `--terra-clinical-data-grid-scrollbar-pinned-column-box-shadow`